css/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+body {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.3rem;
+  color: white;
+  background: url('../background.jpeg') no-repeat fixed;
+  
+
+}
```

Comment:
Please don't leave empty lines within the CSS block this will help keep your code is readable